client/packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -164,13 +187,16 @@ const DetailViewInner = () => {
             <AppBarButtons
               onAddItem={onAddItem}
               simplifiedTabletView={simplifiedTabletView}
+              onUploadDocument={uploadDocumentController.toggleOn}
             />
 
             <Toolbar simplifiedTabletView={simplifiedTabletView} />
 
             <DetailTabs tabs={tabs} />
 
-            <Footer onReturnLines={onReturn} />
+            {tab === InboundShipmentDetailTabs.Details && (
+              <Footer onReturnLines={onReturn} />
+            )}
```

Comment:
document table has its own select + delete actions footer, so hide this one on that tab